docs/tasks/run-application/configure-pdb.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Specifying a Disruption Budget for your Application
+---
+
+{% capture overview %}
+
+This page shows how to limit the number of concurrent disruption
+that your application experiences, allowing for higher availability
+while permitting the cluster administrator to manage the clusters
+nodes.
+
+{% endcapture %}
+
+{% capture prerequisites %}
+* You are the owner of an application running on a Kubernetes cluster that requires
+  high availability.
+* You should know how to deploy [Replicated Stateless Applications](docs/tasks/run-application/run-stateless-application-deployment.md)
+  and/or [Replicated Stateful Applications](docs/tasks/run-application/run-replicated-stateful-application.md).
+* You should have read about the [Pod Disruption Budget concept](docs/tasks/run-application/configure-pdb.md).
+* You should confirm with your cluster owner or service provider that they respect
+  Pod Disruption Budgets.
+{% endcapture %}
+
+{% capture steps %}
+
+## Protecting an Application with a PodDisruptionBudget
+
+1. Identify what application you want to protect with a PodDisruptionBudget (PDB).
+1. Think about how your application reacts to disruptions
+1. Create a PDB definition as a YAML file.
+1. Create the PDB object from the YAML file.
+
+{% endcapture %}
+
+{% capture discussion %}
+
+## Identify an Application to Protect
+
+PDBs can be used with applications the following types of application controllers:
+
+- Deployment
+- ReplicationController
+- ReplicaSet
+- StatefulSet
+
+In future, support may be extended to user-provided controllers ("operators").
+
+Identify the pod selector used by your controller.  You will use this in the
```

Comment:
RW "Identify the pod selector your controller uses. The PodDisruptionBudget uses this to select..."